src/url-replacements.js

```diff
@@ -95,6 +95,17 @@ class UrlReplacements {
       return documentInfoFor(this.win_).pageViewId;
     });
 
+    this.set_('QUERY_STRING', (param, defaultValue = "") => {
+      assert(param, 'The first argument to QUERY_STRING, the query string ' +
+          /*OK*/'param is required');
+      const url = parseUrl(this.win_.location.href);
+      const params = parseQueryString(url.search);
+
+      return (typeof params[param] !== "undefined") ?
```

Comment:
Would probably be better to use `hasOwnProperty` to avoid weird prototype conflicts.